Formula/atkmm.rb

```diff
@@ -18,6 +18,12 @@ class Atkmm < Formula
   depends_on "atk"
   depends_on "glibmm"
 
+  on_linux do
+    depends_on "gcc" => :build
+  end
+
+  fails_with gcc: "5"
```

Comment:
I wonder if `fails_with` can be modified so that it takes a hash where the values can be an array, so that you can do
```
fails_with gcc: ["5", "6"]
```
to avoid having to call it multiple times...